test/functional/outputs/splunk/403_forbidden_test.go

```
@@ -0,0 +1,149 @@
+		found403Metric := false
+		for _, line := range metricLines {
+			// Skip comment lines
+			if strings.HasPrefix(strings.TrimSpace(line), "#") {
+				continue
+			}
+			// Look for vector_http_client_responses_total with status="403"
+			if strings.Contains(line, "vector_http_client_responses_total") &&
+				strings.Contains(line, `status="403"`) {
+				found403Metric = true
+				// Log the metric line for debugging
+				fmt.Fprintf(GinkgoWriter, "Found 403 metric: %s\n", line)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)